### DISCUSSION TITLE
Can't select some folders in library index page

### DISCUSSION THREAD

**Discussion by madtownchris** [photoprism/photoprism#4690] [General]
On the library index page the folder dropdown that lets me select individual folders to re-index doesn't show all folders. 

It's odd because it seems to be selecting a somewhat random set of folders. For example, I have folders named "2024" but the list of available ones goes right from "2020" to the alpha ones skipping 2021, 2022, 2023, etc.

![image](https://github.com/user-attachments/assets/933d1397-058c-4f72-b648-794c0a9f78ec)

**Comment by omerdduran** [comment]
Hello! Are you able to see the folders you need by using the dropdown menu and typing? Additionally, are the photos within folders that are not visible being indexed by PhotoPrism?

**Reply by madtownchris** [reply]
All folders are indexed and no I am unable to type the missing folder names to find those. If, for example, I type "2024" in there, I only get a few folders that happen to have 2024 in the name (like "Florida 2024") not the /2024 folder that I'm looking for.

Also it had somewhat worked previously at some point. By "somewhat" I mean some of the 2024 folders were there but not all of them - I think maybe the root "/2024" was there but not "/2024/2024-10". Not much changed AFAIK and now many are missing.

**Reply by lastzero** [reply]
Could there be a problem with the [filesystem permissions](https://docs.photoprism.app/getting-started/troubleshooting/docker/#file-permissions) that prevents PhotoPrism from accessing these folders?

If they were recently mounted/created, it could also be a caching issue. In this case, [restarting your instance](https://docs.photoprism.app/getting-started/updates/#docker-compose) and reloading the UI might help.

Before you try this, I recommend checking the logs for any errors or warnings as described in our [Troubleshooting Guide](https://docs.photoprism.app/getting-started/troubleshooting/docker/#viewing-logs). 🧐
